src/include/wiredtiger.in

```diff
@@ -7477,8 +7477,15 @@ typedef enum {
     WT_VERBOSE_NOTICE,      /*!< Messages for significant events in WiredTiger, usually worth
                             noting. */
     WT_VERBOSE_INFO,        /*!< Informational style messages. */
-    WT_VERBOSE_DEBUG        /*!< Low severity messages, useful for debugging purposes. */
+    WT_VERBOSE_DEBUG_1,     /*!< Low severity messages, useful for debugging purposes. This is
+                            the default level when debugging. */
+    WT_VERBOSE_DEBUG_2,     /*!< Low severity messages, an increase in verbosity from
+                            the previous level. */
+    WT_VERBOSE_DEBUG_3,     /*!< Low severity messages. */
+    WT_VERBOSE_DEBUG_4,     /*!< Low severity messages. */
+    WT_VERBOSE_DEBUG_5      /*!< Lowest severity messages. */
 } WT_VERBOSE_LEVEL;
+#define WT_VERBOSE_DEBUG WT_VERBOSE_DEBUG_1 /* Temporary, for compatibility. FIXME WT-9925. */
```

Comment:
Super nitpick: But, add a : after the "FIXME".

Reply:
thanks, done in 8c53af4.